ChangeLog.md

```diff
@@ -1,4 +1,49 @@
-﻿## 2017.11.10 Version 5.0.1
+﻿## 2017.11.20 Version 5.1.0
+* Removed AzureRemoteApp Cmdles. Following is the list of removed Cmdlets
```

Comment:
Do not make this change here - this faile is updated by release tools.  Instead, make the change to the Azure module changelog, here: https://github.com/Azure/azure-powershell/blob/preview/src/ServiceManagement/Services/Commands.Utilities/ChangeLog.md